src/SideNavigation/_SideNavList.test.js

```diff
@@ -0,0 +1,74 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { SideNav } from '../';
+
+describe('<SideNavList />', () => {
+    const sideNavList = (
+        <SideNav.List>
+            <SideNav.ListItem
```

Comment:
There should be a `SideNavListItem.test.js` file too.  If we don't have any tests for it, we should at least create the prop spreading test.